tests/Feature/AuthenticationControllerTest.php

```
@@ -0,0 +1,84 @@
+                'token_type',
+            ]);
+
+        $this->assertDatabaseHas('users', [
+            'email' => 'john@example.com',
+        ]);
+    }
+
+    /** @test */
+    public function it_logs_in_a_user_and_returns_a_token()
+    {
+        $user = User::factory()->create([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$user'. (Unused Code Rules)

(UnusedLocalVariable)